arthur_bench/scoring/summary_quality.py

```diff
@@ -59,8 +64,21 @@ class SummaryQuality(Scorer):
     Comprehensive measure of summarization quality compared to a reference summary.
     """
 
-    def __init__(self):
-        self.evaluator = LLMChain(llm=ChatOpenAI(temperature=0), prompt=COMPARE)
+    def __init__(
+        self,
+        llm=ChatOpenAI(temperature=0),
```

Comment:
can you add a type hint for this argument?

Reply:
I am just going with langchain `BaseChatModel` for the class, since as mentioned prompt assumes a chat model. Let me know if I should be using something else.